### ISSUE TITLE
Multiple URL alias' for Tripal Content pages

### ISSUE THREAD

**Issue by laceysanderson** [tripal/tripal#1973]
### Tripal Version

4.x-dev

### Drupal Version

10.4.x-dev

### PostgreSQL Version

16

### PHP Version

8.3

### General Description

We are getting multiple versions of the same URL alias created everytime Tripal Content pages are edited and re-saved. This makes it difficult to maintain these alias' and will eventually cause performance issues.

### Steps to reproduce

1. Create a new Tripal Content page (i.e. Organism), details do not matter.
2. Go to Admin > Configuration > Search and metadata > URL aliases (admin/config/search/path) and search for the alias (i.e. organism/1). On a fresh site, this will be the only alias.
3. Now, edit the Tripal Content page you just created and save it. No changes to the data are needed here. You can even do this multiple times.
4. Finally, go back to the URL aliases page and confirm you now have an entry for each time you edited + the time you saved the page... all with the same alias 🤦 

### Error messages

```shell
There are no errors. This is done silently and would likely go unnoticed until someone tries to make their own alias.
```

### Screenshots

![image](https://github.com/user-attachments/assets/8fefd45e-168d-47b9-b25d-36d1ff159e2b)

**Comment by dsenalik** [comment]
Here is a related Drupal issue describing the same thing
https://www.drupal.org/forum/support/post-installation/2023-03-17/multiple-duplicate-url-aliases-getting-created

It also mentions that the [drush_tools](https://www.drupal.org/project/drush_tools) module has a function to clean up duplicated aliases.

**Comment by dsenalik** [comment]
PR #1980 is closed as this problem will be fixed by PR #1981 in a slightly different way
